client/src/app/site/cinema/components/cinema/cinema.component.ts

```diff
@@ -98,6 +100,15 @@ export class CinemaComponent extends BaseViewComponentDirective implements OnIni
 
     public ngOnInit(): void {
         super.setTitle('Autopilot');
+        this.loadSubscriptions();
+    }
+
+    public async toggleListOfSpeakersOpen(): Promise<void> {
+        await this.listOfSpeakersRepo.setListOpenness(this.listOfSpeakers, this.isLosClosed).catch(this.raiseError);
+    }
+
+    private async loadSubscriptions(): Promise<void> {
+        await this.openslidesService.isStable;
```

Comment:
I'm curious why this await helps here. On the first glance no request is send here, isn't it?

Reply:
You're right. Timeouts, intervals and similar functions have the same result.